src/OpenFeature.Contrib.Providers.Flagd/FlagdProvider.cs

```diff
@@ -28,16 +28,11 @@ namespace OpenFeature.Contrib.Providers.Flagd
     public sealed class FlagdProvider : FeatureProvider
     {
         const string ProviderName = "flagd Provider";
-        static int EventStreamRetryBaseBackoff = 1;
         private readonly FlagdConfig _config;
-        private readonly Service.ServiceClient _client;
-        private readonly Metadata _providerMetadata = new Metadata(ProviderName);
 
-        private readonly ICache<string, object> _cache;
-        private int _eventStreamRetries;
-        private int _eventStreamRetryBackoff = EventStreamRetryBaseBackoff;
+        private readonly Metadata _providerMetadata = new Metadata(ProviderName);
```

Comment:
It's a really good idea to refactor a lot of code out of this file, nice job. There's a lot of unused imports at the top now though.